README.md

```diff
@@ -26,6 +26,11 @@ source venv/bin/activate
 pip install .
 ```
 
+For demonstration : 
+
+`olaf run no_llm_pipeline` to run the no llm_pipeline
+`olaf show no_llm_pipeline` to show all components of the no llm_pipeline
+
```

Comment:
Put this in the section "Quick-start" with the following text : 

Pipelines can be run with the following command: `olaf run demo_pipeline `.
Pipeline components are displayed with the following command: `olaf show demo_pipeline `
The text can be updated in the file `data/demo.txt`.

=> so I would like you to change the command line from `olaf run no_llm_pipeline` to `olaf turn demo_pipeline` and `olaf show no_llm_pipeline` to `olaf show demo_pipeline`